src/profile-logic/profile-data.js

```diff
@@ -2609,15 +2609,23 @@ export function extractProfileFilterPageData(
     return null;
```

Comment:
(optional for this patch)
I think that the name of the function isn't perfect here. It is currently `extractProfileFilterPageData`, but what it does is extract the data to display at the top of the view, like a title. Maybe it could be `extractTitleInformationFromRelevantPages` or something similar?

Reply:
I think it would prefer to do it in a follow-up because it will affect the names in other places as well.